File: 1983/1983.ts
function widestPairOfIndicesWithEqualRa(root: TreeNode | null): number {
    if (!root) return 0;
    return 1 + Math.max(widestPairOfIndicesWithEqualRa(root.left), widestPairOfIndicesWithEqualRa(root.right));
}

File: 1983/1983.cs
public class Solution {
    public int WidestPairOfIndicesWithEqualRa(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(WidestPairOfIndicesWithEqualRa(root.left), WidestPairOfIndicesWithEqualRa(root.right));
    }
}